src/backend/opencl/cl/cn/algorithm.cl

```diff
@@ -24,6 +24,7 @@
 #define ALGO_RX_SFX         0x72151273
 #define ALGO_RX_KEVA        0x7214116b
 #define ALGO_RX_GRAFT       0x72151267
+#define ALGO_RX_XDAG        0x72151208
```

Comment:
Undocumented convention, but the last byte is the ASCII lowercase first letter of the variant name (`s`/`k`/`g`...)

So this should be `0x72151278` because `'x' === 0x78`